internal/servers/repository.go

```diff
@@ -67,18 +75,27 @@ func (r *Repository) listServersWithReader(ctx context.Context, reader db.Reader
 	opts := getOpts(opt...)
 	liveness := opts.withLiveness
 	if liveness == 0 {
-		liveness = defaultLiveness
+		liveness = DefaultLiveness
 	}
+
+	var where string
+	if liveness > 0 {
+		where = fmt.Sprintf("type = $1 and update_time > now() - interval '%d seconds'", uint32(liveness.Seconds()))
```

Comment:
we have a new domain helper function Mike added for Vault that you could use here as:
`update_time > wt_add_seconds_to_now(?)` and then add `-1 * liveness.Seconds()` to the args.